projects/client/src/pages/Login.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useRef } from 'react';
+import toast from 'react-hot-toast';
+import axios from 'axios';
+import { Box, VStack, Heading, Text, FormControl, FormLabel, Input, HStack, Checkbox, Button, Icon, InputGroup, InputRightElement } from '@chakra-ui/react';
+
+import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons';
+import '../App.css'
+export default function Login() {
+
+
+    const [show, setShow] = useState(false)
+    const handleClick = () => setShow(!show)
+
+
+    const email = useRef()
+    const password = useRef()
+
+    let login = async () => {
+        try {
+            let email = email.current.value
+            let password = password.current.value
+
+            let response = await axios.get(`http://localhost:8000/user/login?email=${email}&password=${password}`)
+            toast(response.data.message)
+            localStorage.setItem('myToken', response.data.data.token)
+        } catch (error) {
+            toast(error.response.data.message)
+        }
+    }
+
+    return (<Box w={['full', 'sm']} p={[8, 10]} mt={[20, '15vh']} mx='auto' border={['none', '1px']} borderColor={['', 'gray.500']} borderRadius={10}>
```

Comment:
Wrap in Modal component from Chakra
https://chakra-ui.com/docs/components/modal